crates/runtime/src/instance.rs

```diff
@@ -186,6 +193,16 @@ impl Instance {
                 vmctx: VMContext {
                     _marker: std::marker::PhantomPinned,
                 },
+                #[cfg(feature = "valgrind")]
+                valgrind_state: {
+                    if req.valgrind {
+                        let size = memory_plans.iter().next().map(|plan| plan.1.memory.minimum).unwrap_or(0) * 64 * 1024;
+                        println!("this is the size: {}", size);
```

Comment:
we can remove this println now I think (also search for printlns elsewhere to be sure)